tools/sidekick/config.json

```diff
@@ -1,21 +0,0 @@
-{
```

Comment:
Deleting sidekick config from code since in Helix5 it will be fetched from site configuration json.